sopel/test_tools.py

```diff
@@ -1,12 +1,8 @@
 # coding=utf-8
-"""This module has classes and functions that can help in writing tests.
-
-test_tools.py - Sopel misc tools
-Copyright 2013, Ari Koivula, <ari@koivu.la>
-Licensed under the Eiffel Forum License 2.
-
-https://sopel.chat
-"""
+"""This module has classes and functions that can help in writing tests."""
+# Copyright 2013, Ari Koivula, <ari@koivu.la>
+# Copyright 2019, Florian Strzelecki <florian.strzelecki@gmail.com>
+# Licensed under the Eiffel Forum License 2.
```

Comment:
Ditching the link? 😕

Reply:
Sopel's files are pretty inconsistent regarding that. Beside, I think it's quite redundant: this file will be distributed with Sopel, and it isn't a plugin file.

I believe this link was added to the headers because plugins might be exported, copy and paste here and there. Today, Sopel is a properly distributed package, so I think it's irrelevant to add such link everywhere.

Reply:
Fair enough. Goodness knows we don't want the copyrights or link in there when the Sphinx docs get generated.